pkg/feature/feature.go

```diff
@@ -161,6 +161,15 @@ func (f *Feature) addCleanup(cleanupFuncs ...Action) {
 	f.cleanups = append(f.cleanups, cleanupFuncs...)
 }
 
+func (f *Feature) ApplyManifest(filename string) error {
+	m := loadManifestFrom(filename)
+	if err := m.processTemplate(f.Spec); err != nil {
+		return err
+	}
+
+	return f.apply(m)
+}
```

Comment:
With #797 coming in, especially considering that `fsys` is part of `manifest` struct now, we will need to have a look at how to adjust it @cam-garrison